packages/serverpod/lib/src/server/session.dart

```diff
@@ -341,6 +341,9 @@ class StreamingSession extends Session {
   /// Set if there is an open session log.
   int? sessionLogId;
 
+  /// The endpoint that is currently being processed.
+  String? endpoint;
```

Comment:
Question: Can we have this non-nullable? Or will that mess with the internal session?

Reply:
we can, I have done so in a subsequent change